packages/perseus-editor/src/components/locked-vector-settings.tsx

```diff
@@ -0,0 +1,170 @@
+            }
+        >
+            <View style={[styles.row, styles.spaceUnder]}>
+                {/* Line color settings */}
+                <ColorSelect
+                    selectedValue={lineColor}
+                    onChange={handleColorChange}
+                />
+            </View>
+
+            {/* Zero length error message */}
+            {isInvalid && (
+                <LabelMedium style={styles.errorText}>
+                    {lengthErrorMessage}
+                </LabelMedium>
+            )}
+
+            <LockedFigureSettingsAccordion
+                expanded={tailCoordExpanded}
+                onToggle={() => setTailCoordExpanded(!tailCoordExpanded)}
+                containerStyle={styles.container}
+                panelStyle={styles.accordionPanel}
+                header={
+                    // Summary: Point, coords, color (filled/open)
+                    <View style={styles.row}>
+                        <LabelLarge>{`Tail (${tail[0]}, ${tail[1]})`}</LabelLarge>
+                    </View>
+                }
+            >
+                <CoordinatePairInput
+                    coord={tail}
+                    error={isInvalid}
+                    onChangeProps={(newProps) => {
+                        handleChangePoint(newProps.coord, 0);
+                    }}
+                />
+            </LockedFigureSettingsAccordion>
+
+            <LockedFigureSettingsAccordion
+                expanded={tipCoordExpanded}
+                onToggle={() => setTipCoordExpanded(!tipCoordExpanded)}
+                containerStyle={styles.container}
+                panelStyle={styles.accordionPanel}
+                header={
+                    // Summary: Point, coords, color (filled/open)
+                    <View style={styles.row}>
+                        <LabelLarge>{`Tip (${tip[0]}, ${tip[1]})`}</LabelLarge>
+                    </View>
+                }
+            >
+                <CoordinatePairInput
+                    coord={tip}
+                    error={isInvalid}
+                    onChangeProps={(newProps) => {
+                        handleChangePoint(newProps.coord, 1);
+                    }}
+                />
+            </LockedFigureSettingsAccordion>
+
+            {/* Actions */}
+            <LockedFigureSettingsActions
+                onRemove={onRemove}
+                figureAriaLabel={`locked vector defined by starting point
+                    ${tail[0]}, ${tail[1]} and extending to
+                    ${tip[0]}, ${tip[1]}.`}
+            />
+        </LockedFigureSettingsAccordion>
+    );
+};
+
+const styles = StyleSheet.create({
+    accordionPanel: {
+        paddingBottom: spacing.medium_16,
+    },
+    container: {
+        marginTop: spacing.xSmall_8,
+        marginBottom: 0,
+        marginLeft: -spacing.xxxSmall_4,
+        marginRight: -spacing.xxxSmall_4,
+        backgroundColor: wbColor.white,
+    },
+    errorText: {
+        color: wbColor.red,
```

Comment:
I think you might need a `marginTop` here?

![Screenshot 2024-06-04 at 4 37 47 PM](https://github.com/Khan/perseus/assets/13231763/b0376a16-dae4-4a0d-84c4-b68b9f09ac85)

Reply:
Added.